fingertip/plugins/script.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
+# Copyright (c) 2019 Red Hat, Inc., see CONTRIBUTORS.
+
+import os.path
+import fingertip.exec
+
+
+def main(m, *args, transient=False, no_unseal=False):
+    m.log.debug(f"Plugin: script, args: {args}, transient: {transient}, no_unseal: {no_unseal}")
+    m = m if no_unseal else m.apply('unseal')
+    assert len(args) == 1
+    scriptpath = args[0]
+    m.expiration.depend_on_a_file(scriptpath)
+    with m:
+        m.log.info(f'uploading {scriptpath}')
+        m.ssh.upload(scriptpath)
+        m.log.info(f'uploaded {scriptpath}')
+        scriptname = os.path.basename(scriptpath)
```

Comment:
might be easier to just upload it to some fixed path inside the VM to avoid such problems.